### PR TITLE
v15.0.0

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Thanks you for your contribution to the Azure-SDK-for-Go! We will triage and review it as quickly as we can.

As part of your submission, please make sure that you can make the following assertions:

 - [ ] I'm not making changes to Auto-Generated files which will just get erased next time there's a release.
   - If that's what you want to do, consider making a contribution here: https://github.com/Azure/autorest.go
 - [ ] I've tested my changes, adding unit tests where applicable.
 - [ ] I've added Apache 2.0 Headers to the top of any new source files.
 - [ ] I'm submitting this PR to the `dev` branch, or I'm fixing a bug that warrants its own release and I'm targeting the `master` branch.
 - [ ] If I'm targeting the `master` branch, I've also added a note to [CHANGELOG.md](https://github.com/Azure/azure-sdk-for-go/blob/master/README.md).
 - [ ] I've mentioned any relevant open issues in this PR, making clear the context for the contribution.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -81,737 +137,3 @@
 - Removed the "-beta" suffix.
 - Added various new services.
 - Added ./version package for centralized SDK versioning.
-
-## `v12.5.0-beta`
```
Comment (mcardosos) [inline CHANGELOG.md]: Why are we getting rid of all this?

File: doc.go [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/*
```
Comment (mcardosos) [inline doc.go]: What is this file's purpose?

File: CHANGELOG.md (hunk 1 above)
Comment (jhendrixMSFT) [reply CHANGELOG.md]: Because it clutters up the CHANGELOG with pre-release stuffs.  If people really want this info they can get it from history.

Comment (mcardosos) [reply CHANGELOG.md]: okok

File: doc.go (hunk 2 above)
Comment (jhendrixMSFT) [reply doc.go]: We intend to use it for "introductory" documentation, see https://blog.golang.org/godoc-documenting-go-code.
CC @joshgav

File: profiles/2017-03-09/compute/mgmt/compute/models.go [hunk 3]
```diff
@@ -51,12 +51,20 @@ const (
 	ReadWrite CachingTypes = original.ReadWrite
 )
 
+func PossibleCachingTypesValues() []CachingTypes {
```
Comment (vladbarosan) [inline profiles/2017-03-09/compute/mgmt/compute/models.go]: maybe add in the readme this new feature ?

Comment (jhendrixMSFT) [reply profiles/2017-03-09/compute/mgmt/compute/models.go]: Good point we've had this for so long in latest I forgot about it.

File: doc.go (hunk 2 above)
Comment (mcardosos) [reply doc.go]: Cool :D I didn't know about the doc.go feature

**Pull request merged**